### DISCUSSION TITLE
Image pull latency and unpack latency

### DISCUSSION THREAD

**Discussion by qi0523** [containerd/nerdctl#1570] [Q&A]
How to break down pull latency and unpack latency when using `nerdctl pull` ?

**Comment by fahedouch** [comment]
Do not understand your question, what do you mean by break down pull latency ?

**Reply by qi0523** [reply]
When using `ctr pull`, the console logs the pull latency and unpack latency separately. However, when using `nerdctl pull`, the console only logs the total delay. I want to get the pull latency and unpack latency separately.

**Reply by fahedouch** [reply]
converted to feature ticket https://github.com/containerd/nerdctl/issues/1665
